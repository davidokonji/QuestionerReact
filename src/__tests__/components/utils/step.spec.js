import React from 'react';
import { shallow } from 'enzyme';
import Step from '../../../components/utils/step';

test('should render step component correctly', () => {
  const wrapper = shallow(<Step text="step" />);

  expect(wrapper).toMatchSnapshot();
  expect(wrapper.hasClass('step')).toBeTruthy();
  expect(wrapper.find('p').text()).toBe('step');
});
