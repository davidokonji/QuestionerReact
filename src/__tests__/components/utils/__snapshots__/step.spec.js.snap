// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render step component correctly 1`] = `
<div
  className="step"
>
  <span>
    <img
      alt=""
      height="90px"
      width="70px"
    />
  </span>
  <p
    className="step_text"
  >
    step
  </p>
</div>
`;
